src/index.js

```diff
@@ -12,7 +13,11 @@ export class ArcAds {
     this.positions = [];
     this.collapseEmptyDivs = options.dfp.collapseEmptyDivs;
 
-    window.isMobile = MobileDetection;
+    if (options.desktopBreakpoint) {
+      window.isMobile = new BreakpointDetection(options.desktopBreakpoint);
```

Comment:
This should also have support for tablet/mobile breakpoints in order to be considered